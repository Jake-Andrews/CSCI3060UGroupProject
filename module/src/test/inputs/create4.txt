login
USER
create

USER731
AA
logout
exit