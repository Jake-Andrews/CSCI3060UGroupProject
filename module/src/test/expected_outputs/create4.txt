Welcome, please login to use the program. 
To exit type exit, To start type login: 
Please enter a valid username: 
You have been logged in!

Please enter a command!
For a list of commands type 'help': 
Please enter a valid username: 

Please enter a valid username: 
Please enter a valid usertype: 
Creating the user: USER731 with a usertype of: AA
User created!
Daily transaction file string: 01_USER_____________AA_______________________________0__0.0_____0

Please enter a command!
For a list of commands type 'help': Daily transaction file string: 00_USER_____________AA_______________________________0__0.0_____0

Printing Daily Transactions:01_USER_____________AA_______________________________0__0.0_____0
00_USER_____________AA_______________________________0__0.0_____0
You have been successfully logged out!
Welcome, please login to use the program. 
To exit type exit, To start type login: Thank you for using our program!